RSBypass/RUN_PATCH_RS.command

```diff
@@ -1,4 +1,7 @@
-RS_PATH="/Users/$USER/Library/Application Support/Steam/steamapps/common/Rocksmith2014/Rocksmith2014.app/Contents/MacOS"
+set -e
+RS_PATH="$(/usr/bin/env python3 `dirname $0`/FindRSPath.py)/steamapps/common/Rocksmith2014/Rocksmith2014.app/Contents/MacOS"
```

Comment:
I could change the python-script to return the old default path in case of error.

but: is still relevant with your second comment? if awk is used instead, awk could handle that case, too.

Reply:
It still should go back if it cannot find the file or awk fails for whatever reason